src/shell.js

```diff
@@ -35,8 +35,8 @@ var /** @type {{
 }}
  */ Module;
 if (!Module) /** @suppress{checkTypes}*/Module = {"__EMSCRIPTEN_PRIVATE_MODULE_EXPORT_NAME_SUBSTITUTION__":1};
-#else
-var Module = typeof {{{ EXPORT_NAME }}} != 'undefined' ? {{{ EXPORT_NAME }}} : {};
+#elif !MODULARIZE
```

Comment:
In MODULARIZE mode we can rely on `var Module = config || {};` at the very top of the module code so we don't need to declare `Module` at all here.

Reply:
In non-MODULARIZE we can also do `var Module = Module || {};`.

`var`s get hoisted & coalesced in JavaScript scripts. Since we already declare `var Module`, it's guaranteed to be always defined so the `typeof != 'undefined'` check is not necessary and we can only verify the value.